go/DEVELOPER.md

```diff
@@ -0,0 +1,206 @@
+# Developer Guide
+
+This document describes how to set up your development environment to build and test the GLIDE for Redis Go wrapper.
+
+### Development Overview
+
+The GLIDE for Redis Go wrapper consists of both Go and Rust code. The Go and Rust components communicate in two ways:
+1. using the [protobuf](https://github.com/protocolbuffers/protobuf) protocol.
+2. using shared C objects. [cgo](https://pkg.go.dev/cmd/cgo) is used to interact with the C objects from Go code.
+
+### Build from source
+
+#### Prerequisites
+
+Software Dependencies
+
```

Comment:
I think we should not include instructions what and how to install for glide core. At least not in details.
See #111 and #112

Reply:
I'm a bit confused about what the desired changes would be. I see in #111 you separate the glide-core dependencies into a separate list, which I can do. But the link to glide-core/README.md does not have instructions on how to install these (it looks like it just has recommended VS code extensions), and the dependencies instructions still include commands to download the glide-core dependencies

Reply:
Let me know if you want to jump on a call to clarify